address/Cargo.toml

```diff
@@ -0,0 +1,87 @@
+[package]
+name = "solana-address"
+description = "Solana account addresses"
+documentation = "https://docs.rs/solana-address"
+version = "0.1.0"
+authors = { workspace = true }
+repository = { workspace = true }
+homepage = { workspace = true }
+license = { workspace = true }
+edition = { workspace = true }
+
+[features]
+atomic = ["dep:solana-atomic-u64"]
+borsh = ["dep:borsh", "std"]
+bytemuck = ["dep:bytemuck", "dep:bytemuck_derive"]
+curve25519 = ["dep:curve25519-dalek", "error", "sha2"]
+decode = ["dep:five8", "dep:five8_const", "error"]
+dev-context-only-utils = ["dep:arbitrary", "rand"]
+error = ["dep:num-traits", "dep:solana-program-error"]
+frozen-abi = [
+    "dep:solana-frozen-abi",
+    "dep:solana-frozen-abi-macro",
+    "dep:solana-program-error",
+    "std",
+]
+rand = ["dep:rand", "atomic", "std"]
+sanitize = ["dep:solana-sanitize"]
+serde = ["dep:serde", "dep:serde_derive"]
+sha2 = ["dep:solana-sha256-hasher", "error"]
+std = ["decode"]
+syscalls = ["dep:solana-define-syscall", "error"]
+wasmbind = ["dep:js-sys", "dep:wasm-bindgen"]
+
+[dependencies]
+arbitrary = { workspace = true, features = ["derive"], optional = true }
+borsh = { workspace = true, optional = true }
+bytemuck = { workspace = true, optional = true }
+bytemuck_derive = { workspace = true, optional = true }
+five8 = { workspace = true, optional = true }
+five8_const = { workspace = true, optional = true }
+num-traits = { workspace = true, optional = true }
+rand = { workspace = true, optional = true }
+serde = { workspace = true, optional = true }
+serde_derive = { workspace = true, optional = true }
+solana-atomic-u64 = { workspace = true, optional = true }
+solana-frozen-abi = { workspace = true, features = ["frozen-abi"], optional = true }
+solana-frozen-abi-macro = { workspace = true, features = ["frozen-abi"], optional = true }
+solana-program-error = { workspace = true, optional = true }
+solana-sanitize = { workspace = true, optional = true }
+
+[target.'cfg(not(target_os = "solana"))'.dependencies]
+curve25519-dalek = { workspace = true, optional = true }
+solana-sha256-hasher = { workspace = true, features = ["sha2"], optional = true }
+
+[target.'cfg(target_arch = "wasm32")'.dependencies]
```

Comment:
If we don't make the wasm dependencies optional, it seems they are pulled by default.

Reply:
Resolved since all wasm code was moved to sdk-wasm-js